packages/grid/_modules_/grid/models/gridRows.ts

```diff
@@ -1,10 +1,12 @@
 export type GridRowsProp = Readonly<GridRowData[]>;
-export type GridRowData = { [key: string]: any };
+export type GridRowDefaultData = { [key: string]: any };
```

Comment:
It's better to keep it private than to export it and had to create a breaking change in the future to remove it. 

```suggestion
type GridRowDefaultData = { [key: string]: any };
```